src/stubServer.ts

```diff
@@ -22,10 +22,15 @@ type Response = {
 
 type Route = CommonConfig & Response;
 
+export interface ProxyParams {
+  host: string | undefined;
+  origin: string | undefined;
```

Comment:
origin is already supported and documented or I'm missing something:

```
      POST: {
        headers: { origin: 'https://pmu.fr' },
      }
```

https://github.com/pmu-tech/stub-server/blob/v0.4.0/README.md

Reply:
Indeed,
ProxyParams is now used for internal gesture only (since we have origin & host in headers) but still has "host" optionnal attribute required to change domain for cookies.